### PR TITLE
[#15] Adjust Sign In page design

### PR DESCRIPTION
**Pull request by MaksimPegov**

### CODE REVIEW

File: src/user/login/Login.tsx [hunk 1]
```diff
@@ -151,86 +137,83 @@ export const Login: React.FC = () => {
...
       exit={{ opacity: 0 }}
     >
-      <div className={bem('Header')}>
-        <LoginIcon sx={{ m: 'auto', pr: 1, fontSize: 30 }} />{' '}
-        <div className={bem('HeadeText')}>Autorization</div>
+      <Typography variant="h5" className={bem('Header')}>
+        Sign in
+      </Typography>
+
+      <div className={bem('Body')}>
```
Comment (antonpegov) [inline src/user/login/Login.tsx:144]: Pls refactor it like Cabinet: Header -> Title (and move it inside Grid), move bem() to the Grid container, use same styles as in Cabinet (margins, colours, button width)

File: src/styles/mixins/input.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@mixin input {
+  width: -webkit-fill-available;
+  border: 1px solid $light-gray;
+
+  &_error {
```
Comment (antonpegov) [inline src/styles/mixins/input.scss:5]: Pls use Cammel Case: &_Error

File: src/styles/mixins/input.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    border: 1px solid $red !important;
+  }
+
+  &_disabled {
```
Comment (antonpegov) [inline src/styles/mixins/input.scss:9]: Cammel case

File: src/user/login/Login.tsx [hunk 4]
```diff
@@ -1,44 +1,30 @@
...
 import { onlySpaces } from 'shared/helpers/dataValodation'
 import { userActions } from 'user/state/user.reducer'
-import { loginDataValidation } from 'user/helpers/LoginDataCheck'
-import {
-  selectLogin,
-  selectLoginError,
-  selectRegistrationSuccess,
-  selectSuccessMessage,
-} from 'user/state/user.selectors'
+import { selectLoginError, selectSuccessMessage } from 'user/state/user.selectors'
```
Comment (antonpegov) [inline src/user/login/Login.tsx:20]: Pls align imports

File: src/styles/mixins/input.scss (hunk 2 above)
Comment (MaksimPegov) [reply src/styles/mixins/input.scss:5]: I would like to take conversation about this, in my opinion my type of naming will help us recognise where is classes and where is modificators

**Pull request merged**